tests/index.html

```diff
@@ -10,10 +10,14 @@
       }
     </style>
     <script src="./polyfill.js"></script>
+    <script src="./loader.js"></script>
     <script src="./qunit/qunit.js"></script>
     <script src="/testem.js"></script>
 
     <script type="text/javascript">
+      window.define = define;
+      window.require = require;
```

Comment:
What is even happening here?

Reply:
Good catch, this was from an earlier issue I was trying to solve, it turned out to be something else but I left this here by mistake.